src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/SecondSquare/SecondSquare.tsx

```diff
@@ -80,6 +80,16 @@ const SecondSquare = ({ editPage }: { editPage: boolean }) => {
     }, false);
   };
 
+  const openSingleCluster = () => {
+    const sortedClusters = myAccountStore.ownerAddressClusters?.slice().sort((a: { runWay: number; }, b: { runWay: number; }) => a.runWay - b.runWay);
+    const cluster = sortedClusters.find((c: any) => c.index.toString() === existClusterData.index.toString());
```

Comment:
Looks like its better to filter clusters out, and then sort the result

Reply:
done